client/dist/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+    <head>
+        <title>Client Served</title>
+        <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+        <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
```

Comment:
do you still need these even though you're downloading react?

Reply:
i dont think so, ill remove them later